main.go

```diff
@@ -126,6 +130,10 @@ func run(kubeConfig string) error {
 		}
 	}
 
+	err = enableKubeVirtFeatureGateHostDevices()
```

Comment:
@tlehman we should be able to safely remove this, now that https://github.com/harvester/harvester/pull/2853 has been merged.

Reply:
Done: https://github.com/harvester/pcidevices/pull/15/commits/256641fa90813cb294109c3b2d76d92ea78aab16

Reply:
Eventually we will though, so let's keep it